rfcs/IncrementalDelivery.md

```diff
@@ -0,0 +1,45 @@
+# Incremental Delivery over HTTP
+
+This RFC proposes adding a specification of how "Incremental" GraphQL results should be delivered to clients, using HTTP chunked transfer encoding.
+
+## Incremental Results
+
+An Incremental result is a GraphQL result that is split up into multiple payloads, allowing clients quicker access to parts of the results. Currently this is supported by the proposed `@defer` and `@stream` directives ([RFC](https://github.com/graphql/graphql-spec/blob/master/rfcs/DeferStream.md)).
+
+## `transfer-encoding: chunked`
+
+The HTTP response for an incrementally delivered response should contain the `transfer-encoding: chunked` response header. Chunked transfer encoding allows the body of the response to be delivered as a series of chunks, allowing clients to read each chunk of the response as it is sent without waiting for the entire response.
+
+## `content-type: multipart/mixed`
+
+The HTTP response for an incrementally delivered response should contain the `content-type: multipart/mixed; boundary="-"` response header and conform to the [specification of multipart content defined by the W3 in rfc1341](https://www.w3.org/Protocols/rfc1341/7_2_Multipart.html). An example response body will look like:
```

Comment:
Must the boundary be `"-"` specifically? Could other boundaries be used?

Reply:
I think we should support arbitrary boundaries. [fetch-multipart-graphql](https://github.com/relay-tools/fetch-multipart-graphql) currently only supports `"-"` but I will update that and this document to allow any boundary.

Reply:
@benjie @robrichard Sorry, I didn't notice this comment before merging.
On the last "GraphQL over HTTP" WG we decided on merging and just forgot to actually do it.
Hope it didn't disrupt your workflow and @benjie suggestion can be addressed as separate PR.

Reply:
I've filed an issue here: https://github.com/graphql/graphql-over-http/issues/135 (@IvanGoncharov if you could assign it to @robrichard that would be helpful).